SPECS/grpc/grpc.spec

```
@@ -101,7 +136,17 @@ find %{buildroot} -name '*.cmake' -delete
 %license LICENSE
 %{_bindir}/grpc_*_plugin
 
+
+%files -n python3-grpcio
+%license LICENSE
+%{python3_sitearch}/grpc
+%{python3_sitearch}/grpcio-%{version}-py%{python3_version}.egg-info
+
 %changelog
+* Wed Nov 09 2022 Riken Maharjan <rmaharjan@microsoft.com> - 1.42.0-3
+- Add python binding for gRPC.
+- CBL-Mariner import from Fedora 37 (license: MIT).
```

A small clarification, because I agree that our way of adding specs is confusing at times: this particular wording is reserved for specs we've taken from other distros in whole. This is not a "full" import, we just added the `python3-grpcio` basing on Fedora's spec, so to avoid confusion about the source, let's be clearer on the change introduced in this PR. Here's my suggestion:
```suggestion
- Add 'python3-grpcio' subpackage using Fedora 37 spec for guidance.
```